Sources/SwiftMemcache/MemcachedConnection.swift

```diff
@@ -461,7 +550,7 @@ public actor MemcachedConnection {
     /// - Parameters:
     ///   - key: The key for the value to decrement.
     ///   - amount: The `Int` amount to decrement the value by. Must be larger than 0.
-    /// - Throws: A `MemcachedConnectionError` if the connection to the Memcached server is shut down.
+    /// - Throws: A `MemcachedError` with the code `.connectionShutdown` if the connection to the Memcache server is shut down.
```

Comment:
Let's just put that comment everywhere 
```suggestion
    /// - Throws: A `MemcachedError` that indicates the failure.
```